commit 5bde637bc2c325832a968959d29bb6d3e5834861
Author: bobo1on1 <bob-nospam-@xbmc.org>
Date:   Mon Nov 12 02:52:55 2012 +0100

    fixed: link to pulse-simple, it's needed for CPulseAE::CanInit(), apparently not all distros link libpulse to libpulse-simple

diff --git a/configure.in b/configure.in
index 7889dde..ce40a5b 100644
--- a/configure.in
+++ b/configure.in
@@ -1141,15 +1141,18 @@ if test "x$use_pulse" != "xno"; then
     fi
     USE_PULSE=0
   else
-    AC_CHECK_LIB([pulse], [main],,
-      [if test "x$use_pulse" = "xyes"; then
-        AC_MSG_ERROR($pulse_not_found)
-      else
-        use_pulse=no
-        USE_PULSE=0
-        AC_MSG_RESULT($pulse_not_found)
-      fi])
+    AC_CHECK_LIB([pulse],[main],,pulse_found="no")
+    AC_CHECK_LIB([pulse-simple],[main],,pulse_found="no")
+
+    if test "x$pulse_found" != "xno"; then
       USE_PULSE=1
+    elif test "x$use_pulse" = "xyes"; then
+      AC_MSG_ERROR($pulse_not_found)
+    else
+      use_pulse=no
+      USE_PULSE=0
+      AC_MSG_RESULT($pulse_not_found)
+    fi
   fi
 else
   AC_MSG_RESULT($pulse_disabled)
